### DISCUSSION TITLE
Whats the name of that theme in readme?

### DISCUSSION THREAD

**Discussion by budchirp** [nvim-tree/nvim-tree.lua#1238] [General]
<img src="https://raw.githubusercontent.com/kyazdani42/nvim-tree.lua/master/.github/screenshot4.png">

**Comment by budchirp** [comment]
Thats very unrelated issue but I can't find that theme 😄

**Reply by kyazdani42** [reply]
@reyhankaplan

**Reply by budchirp** [reply]
Thanks/Teşekkürler

**Reply by reyhankaplan** [reply]
Hi @CanKolay3499, it is [material.nvim](https://github.com/marko-cerovac/material.nvim) with my custom overrides.
